.aws/package.json

```diff
@@ -16,7 +16,7 @@
     "node": "=16"
   },
   "dependencies": {
-    "@pocket-tools/terraform-modules": "^3.11.1",
+    "@pocket-tools/terraform-modules": "^3.13.0",
```

Comment:
This adds the `artifactBucketPrefix` parameter in https://github.com/Pocket/terraform-modules/pull/923